.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve InstructLab SDG
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is. -->
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Command Used**
+<!-- Provide the exact `ilab data generate` command and any flags/options used. -->
+
+**Screenshots**
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+**Device Info (please complete the following information):**
+ - Hardware Specs: [e.g. Apple M2 Pro Chip, 16 GB Memory, etc.]
+ - OS Version: [e.g. Mac OS 14.4.1, Fedora Linux 40]
+ - Python Version: [output of `python --version`]
+ - InstructLab Version: [output of `ilab system info`]
```

Comment:
Is this applicable for SDG?

Reply:
Yes, I think this is helpful information while understanding the root cause.